changelog.js

```diff
@@ -1,208 +0,0 @@
-#!/usr/bin/env node
-
```

Comment:
Does this removal have anything to do with switching to Yarn? Looks like a general cleanup.